deployment/deploy-tas.sh

```diff
@@ -54,4 +54,12 @@ cf target \
 
 cp "$CONFIG_DIR/config.js" "$ASSETS_DIR/client/config.js"
 cf push -f "$CONFIG_DIR"/manifest.yml -p "$ASSETS_DIR" --var "api-app-name=$APP_HOST" --var "session-time=$SESSION_TIME"
-cf run-task "$APP_HOST" "ADMIN_EMAIL=$ADMIN_EMAIL ADMIN_PASSWORD=$ADMIN_PASSWORD rake admin:create_user"
+
+CF_VERSION=`cf -v | awk '{print $3}' | awk -F '.' '{print $1}'`
```

Comment:
The other scripts use the POSIX form rather than backticks, so for consistency:
```suggestion
CF_VERSION="$(cf -v | awk '{print $3}' | awk -F '.' '{print $1}')"
```

Reply:
Thank you for your suggestions. I use the POSIX to set CF_VERSION and apply the same fix with deploy-cf.sh.